wandb/sdk/launch/runner/local_container.py

```diff
@@ -108,12 +108,18 @@ def run(
             if not docker_image_exists(image_uri):
                 pull_docker_image(image_uri)
             env_vars.pop("WANDB_RUN_ID")
-            # if they've given an override to the entrypoint
-            entry_cmd = get_entry_point_command(
-                entry_point, launch_project.override_args
-            )
+            entry_cmd = []
+            if entry_point is not None:
+                entry_cmd = entry_point.command
+            override_args = compute_command_args(launch_project.override_args)
             command_str = " ".join(
-                get_docker_command(image_uri, env_vars, entry_cmd, docker_args)
+                get_docker_command(
+                    image_uri,
+                    env_vars,
+                    entry_cmd,
+                    docker_args,
+                    override_args,
```

Comment:
```suggestion
                get_docker_command(
                    image_uri,
                    env_vars,
                    entry_cmd=entry_cmd,
                    docker_args=docker_args,
                    additional_args=override_args,
```
I like doing this for optionals